src/types/notes/interfaces/notes.interfaces.ts

```diff
@@ -0,0 +1,17 @@
+import { CommonEntityFields, Cooperation, UserResponse } from '~/types'
+
+export interface Note extends CommonEntityFields {
```

Comment:
I didn`t see where you are using Note interface, dou you need it except of extending for NoteResponse?

Reply:
I am not using it, removed the interface and edited NoteResponse interface